softwerkskammer/lib/groups/index.js

```diff
@@ -17,7 +18,7 @@ const statusmessage = beans.get('statusmessage');
 const app = misc.expressAppIn(__dirname);
 
 function groupSubmitted(req, res, next) {
-  const group = new Group(req.body);
+  const group = bodyToGroup(req.body);
```

Comment:
Ich verstehe nicht, warum Du das gemacht hast... kannst Du mir das erklären? Das Übersetzen des Parameters wäre doch im Group-Konstruktor sehr gut aufgehoben.

Reply:
Ich mag es wenn meine Konstruktoren dumm sind, Felder zuweisen fertig.
Und in dem Fall finde ich es nicht gut das Details des Auslieferungs Mechanismus Web Form Input in die Gruppe zu packen. Die Logik wir auch nicht gebraucht, wenn man die Gruppe wieder aus der Datenbank ließt. Ich habe dafür gern eigene dedizierte Fabrikmethoden.

Reply:
Ich mag es wenn mein Code gleichförmig ist ;-)

Wie Andreas schon hingewiesen hat, wäre die geeignete Vorlage die Methode fillFromUI aus der Member-Klasse.

Davon abgesehen wäre das alles überhaupt kein Thema, wenn Du die UI schon so designen würdest, dass sie gleich den richtigen Wert schickt. Dann kannst Du die ganze Übersetzungslogik sparen. Das wäre mir die allerliebste Variante.

Reply:
> wenn Du die UI schon so designen würdest, dass sie gleich den richtigen Wert schickt.

Ich freue mich über Vorschläge. Meine Erfahrung mit Forms im Web ist ziemlich begrentzt.